src/json/members/jzwchen.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Joyce Chen",
+  "pennkey": "jzwchen",
+  "major": "Digital Media Design",
+  "school": "Engineering",
+  "bio": "Sophomore in DMD studying computer science graphics and loves spotify group sessions and a sweet treat!",
+  "hometown": "Portland, OR",
+  "team": "Penn Mobile",
+  "roles": ["Designer"],
+  "photo": "https://imgur.com/a/lZqBxFv",
```

Comment:
Looks like something about your image link is causing the build to fail. I think this should be a direct link to the photo (i.e. [https://i.imgur.com/K775b06.jpeg](https://i.imgur.com/K775b06.jpeg)).